HW10_OOP_start/hw/manufacture.py

```diff
@@ -0,0 +1,38 @@
+from animal import Mammal, Bird, Fish, Animal
+
+
+class Anim_manuf:
+
+    def __init__(self, animal_class: str,  **kwargs):
+        self.animal_class = animal_class
+
+        for key, value in kwargs.items():
+            if key == 'name':
+                self.name = value
+            if key == 'age':
+                self.age = value
+            if key == 'color':
+                self.color = value
+            if key == 'voice':
+                self.voice = value
+            if key == 'hair':
+                self.hair = value
```

Comment:
Зачем все это? Просто передайте `kwargs` соответствующему классу. Его конструктор сам разберется.
```suggestion
        self.kwargs = kwargs
```